src/atoms/header-bars/dsmJS/component-test.js

```diff
@@ -7,6 +7,6 @@ suite('dsmJS bar', () => {
   test('that dsmJS is shown', () => {
     const wrapper = shallow(<DsmJsBar />);
 
-    assert.equal(wrapper.dive().text(), 'dsmJS');
+    assert.equal(wrapper.shallow().text(), 'dsmJS');
```

Comment:
i think the `.shallow()` can just be removed here and in the other cases

```suggestion
    assert.equal(wrapper.text(), 'dsmJS');
```

Reply:
Ahh okay! I'm not familiar with the testing platform 🙈 `dive` was giving errors. Thanks :)